README.md

```diff
@@ -59,23 +62,23 @@ Output for 2 events:
 ]
 ```
 
-POST **/events/find** </br>
+POST **/events/by** </br>
 Gets all events that match the given criteria in request body, can be used for any property (name, category, available, points, ...)
 ```
-Input example:
+Example input:
 {
 	"field":"available",
 	"value":true
 }
 
-Output:
+Example output:
 Outputs all events that match the criteria in the same format as GET /events (above).
 ```
 
 GET **/events/id** </br>

```

Comment:
Got a little confused whether `/id` was a part of a resource def or an input :sweat_smile: 
Maybe would be nice to surround with curly or angle brackets like this `/{id}` but not really important :)

Reply:
Same in `put` and `delete` requests maybe :thinking:

Reply:
Yess, glad you noticed it thanks :D I will make it like the others with <>, missed them now